### ISSUE TITLE
Missing info for downloading non-OSM building data

### ISSUE THREAD

**Issue by paulterinho** [ruirzma/osmsc#3]
Hi all, I was curious how to get a hold of the building geojson that is in the `data` folder (in the examples repository)? Is there a URL to download these for cities not in the examples folder?

**Comment by ruirzma** [comment]
Non-OSM building data is downloaded case by case, like [this website](https://gis-pdx.opendata.arcgis.com/) for Portland . 
At present, OSMsc mainly works on OSM data. What I want to demonstrate in the example repository is that OSMsc can also incorporate with non-OSM building data, if any.

**Comment by ruirzma** [comment]
yeah. Recently, I have updated this repo. The previous examples repo is also needed to do so. 
I am glad u have edited the code in __init__.py, I am doing the v.0.2.0 version of it. All these errrors will be fixed.
As for the "key" you mentioned, I guess they are osm keys, instead of access keys (like Mapbox keys). Possibly, the key info in your city object is None, did you add key preferences at the begining? or did OSM return the valid json?
